developer/contributing.rst

```diff
@@ -9,8 +9,8 @@ precious time later.
 Introduce yourself
 ~~~~~~~~~~~~~~~~~~
 
-It won't hurt to join `our main communication channels
-<http://openwisp.org/support.html>`_ and introduce yourself; you can take
+It won't hurt to join `our main communication channel
+<https://gitter.im/openwisp/general>`_ , `dev channel <https://gitter.im/openwisp/development>`_ and introduce yourself; you can take
```

Comment:
I think instead of adding it directly, we should mention that dev channel is for users who are interested in contributing and helping OpenWISP

Reply:
Ohkay! So I should mention ` dev channel for contributors ` ?

Reply:
Yes, you can mention something like that :point_down: 

For those who are interested in development and contribution, join our dev channel

Reply:
Okay, I would follow this. 👍🏻